bats_ai/core/migrations/0024_remove_recording_tag_recording_tags.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.23 on 2025-11-25 15:33
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0023_recordingtag_recording_tag_and_more'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='recording',
+            name='tag',
+        ),
+        migrations.AddField(
+            model_name='recording',
+            name='tags',
+            field=models.ManyToManyField(to='core.recordingtag'),
+        ),
+    ]
```

Comment:
Can you delete this file and 0023 and re-run `docker compose run --rm django ./manage.py makemigrations` so there is only a single migration file?  I didn't think this would have anything specific that requires having two separate migrations.